mesh/mesh.go

```diff
@@ -500,3 +500,66 @@ func (m *Mesh) GetATXs(atxIds []types.AtxId) (map[types.AtxId]*types.ActivationT
 	}
 	return atxs, mIds
 }
+
+// ActiveSetForLayerView - returns the active set size that matches the view of the contextually valid blocks in the provided layer
+func (m *Mesh) ActiveSetForLayerView(layer types.LayerID, layersPerEpoch uint16) (uint32, error) {
```

Comment:
It is not there.
Generally, I rather pass a few params and not make the receiver hold things for me (unless they are used broadly). Mainly because it bloats the reiver and makes testing a bit less comfortable.